src/main/java/com/sdardew/board/domain/post/Post.java

```diff
@@ -0,0 +1,74 @@
+package com.sdardew.board.domain.post;
+
+import com.sdardew.board.domain.post.converter.UserIdConverter;
+import com.sdardew.board.domain.user.User;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+@Entity
+@Table(name = "post")
+public class Post {
+  @GeneratedValue(strategy = GenerationType.AUTO)
+  @Id
```

Comment:
```suggestion
  @Id
  @GeneratedValue(strategy = GenerationType.AUTO)
```

이렇게 쓰는게 전 더 의미가 있더라구요.
아울러 각각의 strategy를 아실까요? 한번 알아보면 어떨까요!?

Reply:
저는 해당 필드가 id인  정보가 더 중요하다고 생각했기 때문에 일부러 필드와 더 가까운 곳에 위치시켰습니다.
혹시 마르코님께서는 왜 해당 방법이 더 의미가 있다고 생각하시는지 여쭤봐도 될까요?
그리고 strategy는 몇개만 기억나는데, 다시 정리를 해보겠습니다!

Reply:
저는 위에서 아래로 읽힌다는 생각으로 제안드렸어요 :)